knowledge_graphs/knowledge_graph_validator.py

```
@@ -114,1319 +118,882 @@
-        if not class_type:
-            return False
+        for method_val in result.method_validations:
+            total_validations += 1
+            confidence_sum += method_val.validation.confidence
+            if method_val.validation.status == ValidationStatus.VALID:
+                valid_count += 1
+            elif method_val.validation.status in [ValidationStatus.INVALID, ValidationStatus.NOT_FOUND]:
+                hallucinations.append(
+                    {
+                        "type": "method",
+                        "element": f"{method_val.method_call.object_type or 'Unknown'}.{method_val.method_call.method_name}",
```

Ruff (0.11.9)

Line too long (125 > 120)

(E501)